script.js

```diff
@@ -1,10 +1,56 @@
-const button = document.querySelector(".Button");
-const title = document.querySelector(".title");
-const author = document.querySelector(".author");
-const booksList = document.querySelector(".books");
+/* eslint-disable no-unused-vars */
 
-const books = [];
+const button = document.querySelector('.Button');
+const title = document.querySelector('.title');
+const author = document.querySelector('.author');
+const booksList = document.querySelector('.books');
 
-function addBook(){
-    const book = {title: ''}
-}
\ No newline at end of file
+const books = JSON.parse(localStorage.getItem('books-list')) || [];
+
+function display() {
+  if (books.length === 0) {
+    booksList.innerText = 'No books in local storage';
+    return 0;
+  }
+  let i = 0;
+  booksList.innerHTML = '';
+  if (books.length !== 0) {
+    books.forEach((book) => {
+      booksList.innerHTML += `<div class='book'>
+    <strong>Title:</strong> ${book.title} <br />
+    <strong>Author:</strong> ${book.author} 
+    <br/><button onclick="remove(${i})"> Remove </button> </div> <br /><hr>
+    `;
+      i += 1;
+    });
+  }
+  return 0;
+}
+
+function remove(i) {
+  const x = document.querySelectorAll('.book')[i];
+  booksList.removeChild(x);
+  books.splice(i, 1);
```

Comment:
- [**Optional**] Let's kindly practice using methods that don't mutate the array like [concat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat), [filter](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter), [slice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) instead of [push](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/push), [pop](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/pop), [splice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice) and so on.

Reply:
@FrederickMih 
Thank you for the feed back. May I ask, what is the reason for practicing methods such as splice() not being recommended! It has shorted code and is faster!

Thank you